content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'endswith()'
+Description: 'Checks whether or not a string ends with the given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `endswith()` string method checks to see if a particular value is attached to the end of the string it is called on and returns a boolean value.
+
+## Syntax
+
+```pseudo
+string.endswith(value, start, end)
+```
+
+## Parameters
+
+- `value` (required): value to be checked at the end of the string
+- `start` (optional): specifies the starting index
+- `end` (optional): specifies the ending index
+
+## Example 1
+
+Checking for the value `g` at the end of the string, `example`:
+
+```py
+example = "This is a string"
+check = example.endswith("g")
+print(check)
+
+# Output: True
+```
+
+## Example 2
+
+Checking for the value `s`:
+
+```py
+example = "This is a string"
+check = example.endswith("s")
+print(check)
+
+# Output: False
+```
+
+## Example 3
+
+Checking for the value `ring`:
+
+```py
+example = "This is a string"
+check = example.endswith("ring")
+print(check)
+
+# Output: True
+```
```

Comment:
I think we should go with just three non-codebyte example. Could we delete lines #55-65 here?